packages/bot-web-ui/src/stores/__tests__/route-prompt-store.spec.tsx

```diff
@@ -0,0 +1,81 @@
+import RoutePromptDialogStore from '../route-prompt-dialog-store';
+
+const mockRouteTo = jest.fn();
+
+const mockCommonStore = {
+    common: {
+        routeTo: mockRouteTo,
+    },
+};
+describe('RoutePromptDialogStore', () => {
+    it('should set should_show to be false', () => {
+        const store = new RoutePromptDialogStore();
```

Comment:
@farabi-deriv can we put this in the before each block? As it is being used in all cases?

Reply:
@shafin-deriv sure. We can do that.